README.md

```diff
@@ -1,7 +1,11 @@
 # Dummy
 
 Dummy makes it clear what elements on the page are "dummy" info (i.e.
-still need to be implemented on the backend).
+still need to be implemented on the backend) by making them visibly different
```

Comment:
`visibly` -> `visually` ?

I don't know if either is correct, it just sounds weird.